### ISSUE TITLE
"Molecule start value" page do not show properly

### ISSUE THREAD

**Issue by jzq90** [Open-Systems-Pharmacology/MoBi#458]
![图片](https://user-images.githubusercontent.com/58142266/77062406-c8dd6a00-6a16-11ea-8d4e-fc235fb50702.png)

**Comment by msevestre** [comment]
Moving it as bug report

**Comment by jzq90** [comment]
> 
> 
> Are you using some kind of font scaling on your system?

No. I don't use any font scaling. Based on your suggestion, I've tried to adjust the font scaling or resolution of the system. But it didn't change the situation.
